Documentation/ApiOverview/Configuration/ConfigurationOverview.rst

```diff
@@ -219,7 +222,8 @@ This is not a complete list of the entire :php:`$GLOBALS` array.
 
    You can find more and view the configuration in the TYPO3 backend
    :guilabel:`SYSTEM > Configuration` (read only) or by viewing the
-   :php:`$GLOBALS` array in a debugger.
+   :php:`$GLOBALS` array in a debugger. The backend module is available with
+   activated lowlevel system extension.
```

Comment:
The word `lowlevel` should be in backticks or similar to indicate that this is a literal name.